android/app/build.gradle

```diff
@@ -135,6 +135,7 @@ android {
 }
 
 dependencies {
+    compile project(':react-native-photo-view')
```

Comment:
Why does this need a native dependency? I feel like it'd probably be best to avoid extraneous native dependencies if possible.